x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/index.tsx

```diff
@@ -318,24 +364,47 @@ export const AllRules = React.memo<AllRulesProps>(
                     </UtilityBarGroup>
                   </UtilityBarSection>
                 </UtilityBar>
-                <MyEuiBasicTable
-                  data-test-subj="rules-table"
-                  columns={columns}
-                  isSelectable={!hasNoPermissions ?? false}
-                  itemId="id"
-                  items={rules ?? []}
-                  noItemsMessage={emptyPrompt}
-                  onChange={tableOnChangeCallback}
-                  pagination={{
-                    pageIndex: pagination.page - 1,
-                    pageSize: pagination.perPage,
-                    totalItemCount: pagination.total,
-                    pageSizeOptions: [5, 10, 20, 50, 100, 200, 300],
-                  }}
-                  ref={tableRef}
-                  sorting={{ sort: { field: 'enabled', direction: filterOptions.sortOrder } }}
-                  selection={hasNoPermissions ? undefined : euiBasicTableSelectionProps}
-                />
+                {tabs}
+                {allRulesTab === AllRulesTabs.rules && (
+                  <MyEuiBasicTable
+                    data-test-subj="rules-table"
+                    columns={columns}
+                    isSelectable={!hasNoPermissions ?? false}
+                    itemId="id"
+                    items={rules ?? []}
+                    noItemsMessage={emptyPrompt}
+                    onChange={tableOnChangeCallback}
+                    pagination={{
```

Comment:
same remarks than @patrykkopycinski for pagination object and you are welcome because I did it and I can tell you it is wrong ;)

Reply:
Thank you ;)